docs/book/src/cronjob-tutorial/testdata/project/controllers/suite_test.go

```diff
@@ -127,22 +126,24 @@ var _ = BeforeSuite(func() {
 		available when talking directly to the API server.
 	*/
 
-	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
+	//TODO: Fix this as it is disabled because following error when tearing down:
```

Comment:
If we add the TODO here then that will appears the docs :-(

Reply:
Test is enabled again